Plugins/Flow.Launcher.Plugin.Shell/Main.cs

```diff
@@ -195,16 +193,18 @@ private ProcessStartInfo PrepareProcessStartInfo(string command, bool runAsAdmin
 
             ProcessStartInfo info = new()
             {
-                Verb = runAsAdministratorArg,
-                WorkingDirectory = workingDirectory,
+                Verb = runAsAdministratorArg, WorkingDirectory = workingDirectory,
             };
             switch (_settings.Shell)
             {
                 case Shell.Cmd:
                     {
                         info.FileName = "cmd.exe";
-                        info.ArgumentList.Add(_settings.LeaveShellOpen ? "/k" : "/c");
-                        info.ArgumentList.Add(command);
+                        info.Arguments = $"{(_settings.LeaveShellOpen ? "/k" : "/c")} {command}";
+                        
+                        // ArgumentList may break original shell command separation with quote.
+                        // info.ArgumentList.Add(_settings.LeaveShellOpen ? "/k" : "/c");
```

Comment:
These commented out code is not removed?

Reply:
yes we can remove it. Probably it would be better to change all argumentlist to plain arguments, because we want to allow user to have the most control over the argument they are writing.

Reply:
yes agree.